app/components/Dashboard/Invest.js

```diff
@@ -113,5 +115,7 @@ Invest.propTypes = {
   estimatePowerUp: PropTypes.func.isRequired,
   handlePowerDown: PropTypes.func.isRequired,
   estimatePowerDown: PropTypes.func.isRequired,
+  modalAdd: PropTypes.func.isRequired,
+  modalDismiss: PropTypes.func.isRequired,
```

Comment:
No need modal actions for invest component (and container). Only place where it used — ExchangeDialog, but it's a container and it's binding modal actions by self

Reply:
OK good catch.